workflows/ampliseq.nf

```diff
@@ -627,6 +630,62 @@ workflow AMPLISEQ {
         ch_versions = ch_versions.mix(SBDIEXPORT.out.versions.first())
     }
 
+    //
+    // MODULE: Create phyloseq objects
+    //
+    if ( !params.skip_taxonomy ) {
```

Comment:
Could you put this into a subworkflow? That would make it easier to maintain, I think?

Reply:
I created a subworkflow and made a few small changes in other parts ampliseq.nf just to make it easier to add the subworkflow.